freqtrade/freqtradebot.py

```diff
@@ -285,8 +292,8 @@ def update_closed_trades_without_assigned_fees(self):
         for trade in trades:
             if trade.is_open and not trade.fee_updated('buy'):
                 order = trade.select_order('buy', False)
-                if order:
-                    logger.info(f"Updating buy-fee on trade {trade} for order {order.order_id}.")
+                open_order = trade.select_order('buy', True)
+                if order and open_order is None:
```

Comment:
i think we'd want to keep the log message there
The check itself seems find i guess - but would we need the same logic for "scaling out" ?

Reply:
I added the log statement back (removed by accident).
The sell logic is checking over already closed trades in which case there would be no changes needed as there shouldn't be any open orders for them.